raw-spans-grouper/src/main/resources/configs/common/application.conf

```diff
@@ -34,14 +36,17 @@ flink.sink {
   log.failures.only = true
   schema.registry {
     schema.registry.url = "http://localhost:8081"
+    schema.registry.url = ${?SCHEMA_REGISTRY_URL}
     value.subject.name.strategy = "io.confluent.kafka.serializers.subject.TopicRecordNameStrategy"
   }
   kafka {
     bootstrap.servers = "localhost:9092"
+    bootstrap.servers = ${?KAFKA_BOOTSTRAP_SERVERS}
   }
 }
 
 span.groupby.session.window.interval = 30
+span.groupby.session.window.interval = ${?SPAN_GROUPBY_SESSION_WINDOW_INTERVAL}
```

Comment:
@buchi-busireddy I was thinking of making default as `10` here, what do you think?

Reply:
Hmm.. I think that should be fine, provided we override it with `30` in SaaS. 

Having said that, this is something the users might need some tweaking (although shouldn't be in a lot of cases) so it might be worth keeping it easily configurable as well.